docs/runtime/_shared-configuration.md

```diff
@@ -109,6 +109,10 @@ Note that `watch` should be enabled for each individual service in the runtime.
 While hot reloading is useful for development, it is not recommended for use in production.
 :::
 
+### `startTimeout`
+
+The number of milliseconds to wait before considering a service as failed to start.
```

Comment:
If I understand the code properly, it's defaulted to `30s`. I think we should add it to the doc 😉

Reply:
You are right! Fixed.